src/HealthCheckServer.ts

```diff
@@ -64,7 +63,7 @@ const defaultHealthHandler = (healthCheck: any): Lifecycle.Method => {
  * @description the failure handler for Hapi. We put this here to make it more testable
  *
  */
-const failAction = async (_request: Hapi.Request, _handler: Hapi.ResponseToolkit, err?: Error) => {
+const failAction = async (_request: Hapi.Request, _handler: Hapi.ResponseToolkit, err?: Error): Promise<void> => {
   if (!err) {
     throw Boom.boomify(new Error(`Unknown Server Error`))
```

Comment:
Why are you using boom errors here instead of the error handling framework

Reply:
Will fix this.